project_0a/project_0a_6.c

```diff
@@ -0,0 +1,40 @@
+/*
+Project 0a
+*/
+
+/*
+Exercise 5 - build a multiplication table for a given number. Use scanf() to
+read input from keyboard. Limit previous implementation (exercise 5) to not
+allow negative values nor positive higher than 50
+
+Note: if statement is represented as:
+if (confition) {code for true condition} else {code for false condition}
+The else bit is optional
+*/
+
+#include <stdio.h>
+
+int main() {
+  int number, factor, calculation;
+
+  printf(
+      "\nFor which number would you like to see the multiplication table?\n");
+  scanf("%d", &number);
+
+  if (number < 0 || number > 50) {
```

Comment:
Prefer early termination rather than placing the rest of the program inside the `else` block. This reduces nesting and makes the code more readable. Example:

```
if(...){
    // Add your error message.
    return 1;
}
// Rest of the code...
```

Reply:
Deleted `else` block and changed code as requested

Reply:
```
if(...){
    // Add your error message.
    return 1;
    // Rest of the code...
    return 0;
}
```

This is what is currently implemented, which is not exactly what I meant.